autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -57,6 +57,16 @@ def setUpClass(cls):  # nopep8 - ignore N802
         """
         super(AutoscaleRackConnectFixture, cls).setUpClass()
 
+        if cls.rcv3_client is None:
+            # don't bother with the rest of the set-up; skip all tests
+            skip = unittest.skip("This account does not support RCv3.")
+
+            for test in inspect.getmembers(cls, predicate=inspect.ismethod):
+                # test is a tuple of name, object
+                if test[0].startswith("test_"):
```

Comment:
You can call unittest.skipTest() during `setUp`, apparently, but it didn't seem to work in `setUpClass` either.  :|

I have an idea of how to skip the whole class, but it involves a lot more refactoring.  I guess let me try that...